tesseract/integration-test/src/lib.rs

```diff
@@ -126,33 +86,13 @@ async fn relay_get_response_message(
 	};
 	// =================== send to the source chain ================================
 	let _res = chain_a_client.submit(vec![response]).await?;
-	//==================== after approx 7-9 blocks the response event is emitted ===
 	// =================== fetch the returned value ================================
+	let encoded_event = chain_a_sub_client
```

Comment:
This can't be a reliable way to get the events; the event could have been emitted before this function is executed; instead of using events, you can also modify the demo pallet to store the result of the get response so you can just read that and use it to compare.